streams/README.txt

```diff
@@ -0,0 +1,8 @@
+The source code for these examples live in the corresponding
```

Comment:
How about:

The source code for most of these examples is under web/ directories
but could just as easily be under bin/ directories, because most of these
examples don't rely on dart:io.

The exception is read_file, which relies on dart:io.